modules/sdk/project-templates/project-templates/src/test/java/com/liferay/project/templates/ProjectTemplatesTest.java

```diff
@@ -4340,6 +4371,21 @@ private static File _testNotExists(File dir, String fileName) {
 		return file;
 	}
 
+	private static void _testPropertyExists(Path path, String property)
+		throws Exception {
+
+		try (Stream<String> stream = Files.lines(path)) {
```

Comment:
It's just a test, wouldn't it be easier to open the file with `Properties` and check if the map contains the property? cc @petershin

Reply:
Agreed!